src/analysis/transcendental/liouville_constant.lean

```diff
@@ -23,9 +23,39 @@ open set real finset
 
 variable (m : ℕ)
 
-local notation `α` := ∑' (i : ℕ), 1 / (m : ℝ) ^ i!
-local notation `α_first` k `terms` :=  ∑ i in range (k+1), 1 / (m : ℝ) ^ i!
-local notation `α_terms_after` k := ∑' i, 1 / (m : ℝ) ^ (i + (k+1))!
+/--
+liouville constant is
+∞
```

Comment:
Our online docs support LaTeX; you can surround the formula in `$` and `$` for an inline formula or `$$` and `$$` for a display mode expression.

If you do want to use ASCII art, then you should surround it with lines that contain only ` ``` ` so that Markdown will display them as code blocks.

Reply:
Thank you. I have changed it to LaTeX.